### DISCUSSION TITLE
Should `SomeStrEnum(str, Enum)` require `SLOT000`?

### DISCUSSION THREAD

**Discussion by jamesbraza** [astral-sh/ruff#5747] [Q&A]
Please see the below file with Python 3.10.10:

```python
from enum import Enum

class SomeStrEnum(str, Enum):
    KEY = "value"
```

Running `ruff==0.0.278` on this:

```bash
> ruff --isolated --select="SLOT" a.py
a.py:3:7: SLOT000 Subclasses of `str` should define `__slots__`
Found 1 error.
```

Running `flake8-slots==0.1.5` with `flake8==4.0.1` on this, there's no error:

```bash
> flake8 a.py
```

Fwiw, Python 3.11's built-in [`enum.StrEnum`](https://docs.python.org/3/library/enum.html#enum.StrEnum) doesn't seem to define `__slots__`.

I am not sure if there should or should not be `SLOT000` for this.  Looks like @qdegraaf added `SLOT` rule in `ruff==0.0.273`, so notifying him.

I am thinking either:
- This is a false positive of `SLOT000`
- Or should the built-in `StrEnum` define `__slots__`?

**Comment by charliermarsh** [comment]
Looks like a bug, I converted to an issue #5748/
